### DISCUSSION TITLE
Frame-by-frame seeking does not work correctly on scrolling rulesets (potentially due to autoplay generator)

### DISCUSSION THREAD

**Discussion by frenzibyte** [ppy/osu#26713] [Ideas]
### Type

Crash to desktop

### Bug description

-

### Screenshots or videos

https://github.com/ppy/osu/assets/22781491/0e09e964-3d2e-4485-a5a0-bdd898a2b4f0

(same applies to catch and mania)

### Version

baaf33d995caa9fd5ea491592fcae39b4140874c

### Logs

-

**Comment by peppy** [comment]
This is intended. I'm not sure why this is a "crash to desktop".

Sure maybe we want to rewrite the auto generators to generate more frames, but the idea of the frame controls is to expose the underlying replay in its raw state.
